tests/test_cnn/test_crf.py

```diff
@@ -0,0 +1,46 @@
+import itertools
+import torch
+
+from mmcv.cnn.bricks import ConditionalRandomField
+
+
+def compute_score(crf, emission, tag):
+    # emission: (seq_length, num_tags)
+    assert emission.dim() == 2
+    assert emission.size(0) == len(tag)
+    assert emission.size(1) == crf.num_tags
+    assert all(0 <= t < crf.num_tags for t in tag)
+
+    # Add transitions score
+    score = crf.start_transitions[tag[0]] + crf.end_transitions[tag[-1]]
+    for cur_tag, next_tag in zip(tag, tag[1:]):
+        score += crf.transitions[cur_tag, next_tag]
+
+    # Add emission score
+    for emit, t in zip(emission, tag):
+        score += emit[t]
+    return score
+
+
+def test_ConditionalRandomField():
```

Comment:
Please refer to https://github.com/kmkurn/pytorch-crf/blob/master/tests/test_crf.py to improve the unit test